src/main/java/org/openpnp/gui/MachineControlsPanel.java

```diff
@@ -272,7 +272,7 @@ public void actionPerformed(ActionEvent arg0) {
 					Configuration.get().getMachine().setEnabled(enable);
 					setEnabled(true);
 					if (machine.getHomeAfterEnabled() && machine.isEnabled()) {
-						selectedTool.getHead().home();
```

Comment:
okay, changed the gui elements to perform a homing cycle on the machine instead directly on the head. with the nulldriver it works as expected now.